cmd/storer2elastic/dataprocessor/ratingsProcessor.go

```diff
@@ -160,11 +164,17 @@ func (rp *ratingsProcessor) createPeerAdapter() error {
 		return err
 	}
 
+	storagePruning, err := rp.getNewStoragePruningManager(pathManager)
```

Comment:
please update your feature branch. This app was removed.

Reply:
Updated, but with the master branch, in which this app is still present.